compute-budget-instruction/src/compute_budget_instruction_details.rs

```diff
@@ -232,6 +236,10 @@ mod test {
         solana_instruction::Instruction,
         solana_keypair::Keypair,
         solana_message::Message,
+        solana_program_runtime::execution_budget::{
+            DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT, MAX_BUILTIN_ALLOCATION_COMPUTE_UNIT_LIMIT,
+            MAX_COMPUTE_UNIT_LIMIT, MAX_HEAP_FRAME_BYTES, MIN_HEAP_FRAME_BYTES,
+        },
```

Comment:
aren't these imported in the super?

Reply:
Yes, looks like my IDE automatically imported these. I'll remove.